builds/jazz-build-module/sls-app/sbr.groovy

```diff
@@ -1121,17 +1123,24 @@ def prepareServerlessYml(aConfig, env, configLoader, envDeploymenDescriptor) {
   }
   try {
     def appContent = readFile('application.yml').trim()
-    if(!appContent.isEmpty()) deploymentDescriptor = appContent
-    } catch(e) { // TODO to catch the type error
-      echo "The application.yml does not exist in the code. So the default value from config will be used.$e"
+    if(!appContent.isEmpty()) {
+      echo "The application.yml is being used."
```

Comment:
Now that user is not aware of application.yml, should this be `user supplied serverless.yml is being used`?